config/types.go

```diff
@@ -7,15 +7,26 @@ import (
 	"github.com/hashicorp/consul/api"
 )
 
-// ServiceTokenFilename is the file in the BootstrapDir where the service token is written by `consul login`.
-const ServiceTokenFilename = "service-token"
+const (
+	// ServiceTokenFilename is the file in the BootstrapDir where the service token is written by `consul login`.
+	ServiceTokenFilename = "service-token"
 
-// ClientTokenFilename is the file in the BootstrapDir where the Consul client token is expected.
-// The consul-ecs binary does not write this file, but health-sync will attempt to do a `consul logout` for this token.
-const ClientTokenFilename = "client-token"
+	// ClientTokenFilename is the file in the BootstrapDir where the Consul client token is expected.
+	// The consul-ecs binary does not write this file, but health-sync will attempt to do a `consul logout` for this token.
+	ClientTokenFilename = "client-token"
 
-// DefaultAuthMethodName is the default name of the Consul IAM auth method used for `consul login`.
-const DefaultAuthMethodName = "iam-ecs-service-token"
+	// DefaultAuthMethodName is the default name of the Consul IAM auth method used for `consul login`.
+	DefaultAuthMethodName = "iam-ecs-service-token"
+
+	// DefaultGatewayPort (8443) is the default gateway registration port used by 'consul connect envoy -register'.
+	DefaultGatewayPort = 8443
```

Comment:
Given Consul ECS's architecture, I wonder if there will ever be a good reason to override any of these ports 🤔

Reply:
:thinking: I'm not sure that the question is if there is a **_good_** reason to do it.. but at some point someone probably **_will_** want to override it :wink: I think my preference would be to leave it defaulted for now to keep it simple.. we could make it configurable later. Thoughts/Objections?

Reply:
k8s defaults the pod bind port to 8443 but the external port to 443 iirc